frontend/src/types/LogRecordTypes.ts

```diff
@@ -23,6 +23,27 @@ export type GetLogRecordCountResponse = {
 } | null;
 
 export type PostLogRecordsResponse = Omit<
-LogRecord, 
-'logId' | 'attnToFirstName' | 'attnToLastName' | 'employeeFirstName' | 'employeeLastName' | 'tags'
+  LogRecord,
+  | "logId"
+  | "attnToFirstName"
+  | "attnToLastName"
+  | "employeeFirstName"
+  | "employeeLastName"
+  | "tags"
 > | null;
```

Comment:
nit: Might be easier / cleaner to convert this to a `pick` type and only include the fields we're concerned with